### PR TITLE
Alpha Branch Using Beta Wrapper

### PR DESCRIPTION
**Pull request by herbertmilhomme**
## Status

- [ ] **Ready**
- [x] **In development** 
- [ ] **Hold**

### CODE REVIEW

**Comment by herbertmilhomme** [comment]
The merge has been completed and the alpha branch can be used for development. There's no longer any reason to keep this branch as a pending PR.

File: .gitignore [hunk 1]
```diff
@@ -74,6 +74,8 @@ git-journal.fish
 [Uu]nit[Tt]est[Pp]roject/*.dll
 [Uu]nit[Tt]est[Pp]roject/*.xml
 */bin/*
+/Pokemon Unity/*.sln
```
Comment (gustavogalo781-prog) [inline .gitignore:77]: Sanchez

Comment (gustavogalo781-prog) [reply .gitignore:77]: Sanches

Comment (gustavogalo781-prog) [reply .gitignore:77]: Bosta

Comment (gustavogalo781-prog) [reply .gitignore:77]: Sanches

Comment (gustavogalo781-prog) [reply .gitignore:77]: GusTAVO

**Comment by gustavogalo781-prog** [comment]
Elétrico

**Pull request merged**